### PR TITLE
Add and test support for null values

### LINKED ISSUE #9: `null` not supported
When the state includes a `null` value, valtio-yjs throws an error:

```
Uncaught Error: unsupported p type
```

I encountered this when trying to synchronize an array of excalidraw elements between clients, which often contain a null boundElementIds property.

Possibly the cause of #6

### PR DESCRIPTION
**Pull request by mattrossman**
Resolves #9

Feel free to simplify the test case.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -50,6 +51,7 @@ export const bindProxyAndYMap = <T>(p: Record<string, T>, y: Y.Map<T>) => {
       bindProxyAndYMap(pv, yv);
       p[k] = pv as unknown as T;
     } else if (
+      yv === null ||
```
Comment (dai-shi) [inline src/index.ts:54]: Is this covered by the test?

**Review by dai-shi — COMMENTED** [review]
Cool. the test looks good too.

Do we want to do the same for `bindProxyAndYArray`??

**Comment by mattrossman** [comment]
> Do we want to do the same for bindProxyAndYArray??

It seems `Y.Array` doesn't work with null values. When I try to do this:
```js
const a = doc.getArray('arr')
a.push([null])
```

I get the error `TypeError: Cannot read property 'constructor' of null` from this line in yjs:

https://github.com/yjs/yjs/blob/df9bfbe778104233db06e970763bfb931f6c4a60/src/types/AbstractType.js#L649

File: src/index.ts (hunk 1 above)
Comment (mattrossman) [reply src/index.ts:54]: I believe so. Without that line, I get the error `unsupported y type`

Comment (dai-shi) [reply src/index.ts:54]: Cool. Thanks for the confirmation.

**Comment by dai-shi** [comment]
> It seems `Y.Array` doesn't work with null values. When I try to do this:

Interesting. Thanks for confirming this too!

**Review by dai-shi — APPROVED** [review]
Thanks for your contribution!

**Comment by dai-shi** [comment]
Published: https://www.npmjs.com/package/valtio-yjs/v/0.1.2
